### DISCUSSION TITLE
NEED --FOLLOW IN HISTORY !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by qs-sbrunner** [community/community#6964] [Other Feature Feedback, Questions, & Ideas] [answered]
Quoted from Harel M.:

Please add an option to use the --follow option in the settings somewhere or enable it by default.
This issue is open for a decade now...

Could not say it better.

**Comment by davidstaheli** [comment]
Thanks for raising this to the top of the stack again, @qs-sbrunner. We'll re-prioritize this in our 2022 planning.

**Reply by thehans** [reply]
@davidstaheli 
***Did*** this issue end up being prioritized for 2022?

I can find no mention of it on the [public roadmap](https://github.com/orgs/github/projects/4247/views/1)

When can we expect an update of any sort on this issue?

**Reply by Alnaimi-** [reply]
This has been a requested feature since 2009

**Reply by davidstaheli** [reply]
This hasn't yet been scheduled, but we're tracking it on our backlog as something we want to do this year. A few months ago, we arranged for additional capacity to address items like this that have waited for so long. Now that additional capacity is available, it's just a matter of scheduling based on relative priority. We're anxious to get this one done, and I hope to soon have a clearer date to post here.

**Comment by swharden** [comment]
Thanks for raising this @qs-sbrunner! I'm looking forward to this feature being added 🤞 

Related,
* #3272

**Comment by davidstaheli** [comment, marked as answer]
Thanks to @shmeadyy and team, this feature shipped today! 🎉 We'd love to hear any feedback/suggestions. More details are here: https://github.blog/changelog/2022-06-06-view-commit-history-across-file-renames-and-moves/

**Comment by sakamati** [comment]
Is this available on GitHub Enterprise?

**Reply by gegles** [reply]
Does anybody know which version of `GitHub Enterprise` this is available (if any)? We're still on 3.3 and will upgrade to only 3.5 next week (here at IBM).
